src/StaticHelper.cs

```diff
@@ -0,0 +1,332 @@
+
+        /// <summary>
+        /// Получить функциональный текст из функции
+        /// </summary>
+        /// <param name="function">Функция</param>
+        /// <returns></returns>
+        public static string GetFunctionalText(Function function)
+        {
+            string functionalText = function.Properties.FUNC_TEXT_AUTOMATIC
+                .ToString(ISOCode.Language.L___);
+            
+            if (string.IsNullOrEmpty(functionalText))
+            {
+                functionalText = function.Properties.FUNC_TEXT_AUTOMATIC
+                    .ToString(ISOCode.Language.L_ru_RU);
+            }
+            if (string.IsNullOrEmpty(functionalText))
+            {
+                functionalText = function.Properties.FUNC_TEXT
+                    .ToString(ISOCode.Language.L_ru_RU);
+            }
+            if(string.IsNullOrEmpty(functionalText))
+            {
+                return "";
+            }
+
+            return functionalText;
+        }
+
+        /// <summary>
+        /// Возвращает имя канала (IO-Link, DI, DO) из строки для IO-Link
+        /// модуля.
+        /// </summary>
+        /// <param name="source">Строка для поиска</param>
+        /// <returns></returns>
+        public static string GetChannelNameForIOLinkModuleFromString(
+            string source)
+        {
+            const string IOLink = "IO-Link";
+            const string DI = "DI";
+            const string DO = "DO";
+
+            if (source.Contains(DI) &&
+                !source.Contains(IOLink) &&
+                !source.Contains(DO))
+            {
+                return DI;
+            }
+
+            if (source.Contains(DO) &&
+                !source.Contains(IOLink) &&
+                !source.Contains(DI))
+            {
+                return DO;
+            }
+
+            return "";
+        }
+
+        /// <summary>
+        /// MatchEvaluator для regular expression,
+        /// замена русских букв на английские
+        /// </summary>
+        public static string RussianToEnglish(Match m)
+        {
+            switch (m.ToString()[0])
+            {
+                case 'А':
+                    return "A";
+                case 'В':
+                    return "B";
+                case 'С':
+                    return "C";
+                case 'Е':
+                    return "E";
+                case 'К':
+                    return "K";
+                case 'М':
+                    return "M";
+                case 'Н':
+                    return "H";
+                case 'Х':
+                    return "X";
+                case 'Р':
+                    return "P";
+                case 'О':
+                    return "O";
+                case 'Т':
+                    return "T";
+            }
+
+            return m.ToString();
+        }
+    }
+
+    /// <summary>
+    /// Класс содержащий общие константы  для проекта.
+    /// </summary>
+    public static class ConstVars
```

Comment:
Может класс назвать по-другому?
```suggestion
    public static class CommonConst
```